assets/js/components/notifications/UnsatisfiedScopesAlert.js

```diff
@@ -99,6 +103,13 @@ export default function UnsatisfiedScopesAlert() {
 	let moduleNames;
 	// Determine if all scopes are in Google API format, otherwise use generic message.
 	if (
+		gteSupportEnabled &&
+		unsatisfiedScopes.includes(
+			'https://www.googleapis.com/auth/tagmanager.readonly'
+		)
```

Comment:
As per the AC:

> _If_ the `gteSupport` feature flag is enabled _and_ the user has Analytics active _and_ the user does not have Tag Manager active _and_ the only scope that the user has not granted is `https://www.googleapis.com/auth/tagmanager.readonly`:


...this condition needs to be tightened up to check that `unsatisfiedScopes.length === 1`. It could also check via the indexed array rather than `includes`.

```suggestion
		unsatisfiedScopes.length === 1 &&
		unsatisfiedScopes[ 0 ] === 'https://www.googleapis.com/auth/tagmanager.readonly'
```